rfc/rfc014-authorization-credential.md

```diff
@@ -101,37 +102,39 @@ When `implied`, values MUST be added to the `restrictions` array.
 
 The `id` field MUST contain the DID of the actor.
 The `subject` field MAY contain the patient identifier. In the example above the *oid* for the Dutch citizenship number is used.
-The `service` field refers to a Bolt. A Bolt MUST describe the set of FHIR resources that can be accessed with the credential.
+The `purposeOfUse` field refers to a Bolt. A Bolt MUST describe the set of FHIR resources that can be accessed with the credential.
```

Comment:
5.2.1.9 states that

> It contains a list of policy names.

So does it refer to a Bolt, policies defined by the Bolt or policies referred to by the Bolt?

Reply:
changed it to refer to a policy defined by a Bolt.